lighthouse-core/config/default.json

```diff
@@ -2,6 +2,7 @@
   "passes": [{
     "recordNetwork": true,
     "recordTrace": true,
+    "emulation": "iphone4",
```

Comment:
I added extra empty passes at first, but for some sites this adds a really long time. Not sure if these emulation changes will break anything